subprojects/plugins/src/integTest/groovy/org/gradle/java/compile/IncrementalCompileMultiProjectTestFixture.groovy

```diff
@@ -42,15 +43,15 @@ trait IncrementalCompileMultiProjectTestFixture {
         file('app/src/main/java/AClass.java') << 'public class AClass { }'
     }
 
-    def getAppCompileJava() {
+    static String getAppCompileJava() {
```

Comment:
I think we shouldn't use static methods on `trait`s: http://docs.groovy-lang.org/next/html/documentation/core-traits.html#_static_methods_properties_and_fields

Reply:
Good point, removed the `static` modifiers.